### ISSUE TITLE
Can it support the file type of makefile?

### ISSUE THREAD

**Issue by 853419196** [formulahendry/vscode-code-runner#154]

**Comment by formulahendry** [comment]
You could customize makefile support in "code-runner.executorMap" in settings.json. I am not familiar with makefile. Is that a general command? If it is, I could add it into the extension.

**Comment by JeffreyCA** [comment]
Yes, I was wondering too. The filename is always called `Makefile` (no extension), and you usually execute a single `make` command.

I tried the following:

```
"code-runner.executorMap": {
        ...
        "Makefile": "make"
}
```

but it is not working for me. Right now I just exceute other source files (.cpp) and call `make` through `executorMap` on .cpp files.

**Comment by jendrikw** [comment]
The language identifier is `makefile` (lowercase), see https://code.visualstudio.com/docs/languages/identifiers#_known-language-identifiers.

Also, make needs `-f` before the path to the makefile.

This works for me:

```
"code-runner.executorMap": {
    "makefile": "make -f"
},
```
